### ISSUE TITLE
Hosted fields: New cardholderName input shows numeric keypad

### ISSUE THREAD

**Issue by martinandert** [braintree/braintree-web#523]
### General information

* SDK version: 3.65.0
* Environment: any
* Browser and OS: any

### Issue description

Here's how the field is rendered in the iframe:

```html
<input
  autocomplete="cc-name" 
  type="text" 
  autocorrect="off" 
  autocapitalize="none" 
  spellcheck="false" 
  class="cardholderName" 
  data-braintree-name="cardholderName" 
  name="cardholder-name" 
  id="cardholder-name" 
  pattern="\d*" 
  maxlength="255" 
  aria-describedby="field-description-cardholderName"
>
```

I guess it's the `pattern="\d*" ` attribute what is causing the numeric keypad to appear on mobile.

**Comment by crookedneighbor** [comment]
Yep, you are correct. We'll get this fixed up. Thanks for the report!

**Comment by crookedneighbor** [comment]
Fixed in v3.66.0 (note that [npm is currently having issues publishing packages](https://status.npmjs.org/incidents/13n2894dytyt) ), so this is available via our CDN, but may take a while to be available on npm.
